### PR TITLE
Mobile

### PR DESCRIPTION
**Pull request by MarcoOquendoC**
# [Portfolio: mobile version](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)

### These are the changes proposed to the Portfolio project:

- Add the "About myself" section.
- Grid in the main container that contains all of the cards.
- Flexbox to lay out an individual card.
- Use **ul** and **li** for categories in "Work" section.
- Use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.
- Implement the button interactions (enable, hover, etc.).
- Update README.md file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,34 +29,168 @@ <h2 class="title ocupation">
...
         </div>
       </section>
+      <section class="works">
+        <h2 class="section-title">
+          My Recent Works
+        </h2>
+        <div class="card-container">
+          <div class="card">
+            <div class="card-img">
+              <p>img</p>
```
Comment (koachcode) [inline index.html]: - Please remove the `<p>img</p>` as it shows on the image inserted for the project, which is not required in this project.
- PLease the text `see project` and `Get My Resume` should have a color of  `#fff` as it is in the Figma design, currently yours is having a color of black. please fix it to use the appropriate colors.
- PLease for the `Multipost story...` the color should be `color: #3A4A42;` as it is in the figma design. 

##### Figma design 
Screenshot from 2022-07-02 16-05-49

##### Your design 
Screenshot from 2022-07-02 16-09-23

File: index.html [hunk 2]
```diff
@@ -29,34 +29,168 @@ <h2 class="title ocupation">
...
+        <div class="about-container">
+          <h2>
+            About me
+          </h2>
+          <img src="/images/about.svg" alt="about">
+        </div>
+        <p>
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <img class="about2" src="/images/about2.svg" alt="about2">
```
Comment (koachcode) [inline index.html]: - Please make this a background image with the CSS property background-image and position appropriately,  as it is not a main part of the portfolio but is for decoration. Do the same for other images that are for decoration or background-images.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @MarcoOquendoC , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- NO linter errors :+1: 
- Good PR summary message :+1: 
- Good commit messages :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MarcoOquendoC** [comment]
# Hi @Ginohmk,

## Thanks for your kindness. The issues are corrected!


### Required corrections:
> - Remove the p tag as it shows on the image inserted for the project, which is not required in this project.
> - Change the text color to "See Project" and "Get My Resume" lines to  #fff as it is in the Figma design.
> - Change "Multi-Post Stories Gain+Glory" text color to #3a4a42.
> - Make background images with the CSS property background-image and appropriate position instead of html img tags.

**Review by danielufeli — APPROVED** [review]
Hi @MarcoOquendoC,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**